core/templates/services/audio-bar-status.service.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about status of AudioBar
```

Comment:
Can you explain why this was required? (I wanted to know this so that we don't remove this in future unexpectedly.) 

[Maybe you can add this in the file overview as well because I wasn't able to understand the major requirement fo this service.]

Reply:
Done